stfl/itemview.stfl

```diff
@@ -1,4 +1,4 @@
-vbox
+!vbox
```

Comment:
Same issue as commented on help.stfl.
Here, it also looks like it does work correctly if the focus (exclamation mark) is put in front of the `textview[article]` line